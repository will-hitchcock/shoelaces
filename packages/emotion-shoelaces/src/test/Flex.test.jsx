import React from 'react';
import { shallow } from 'enzyme';
import { Flex } from '..';

describe('<Flex />', () => {
  it('should render a <Flex />', () => {
    expect(shallow(<Flex />)).toMatchSnapshot();
  });
});
