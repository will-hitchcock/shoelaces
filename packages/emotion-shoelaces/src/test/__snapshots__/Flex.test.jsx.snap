// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex /> should render a <Flex /> 1`] = `
<styled.div
  _css={
    Object {
      "alignContent": "flex-start",
      "alignItems": "stretch",
      "alignSelf": "auto",
      "display": "flex",
      "justifyContent": "space-between",
    }
  }
/>
`;
